### ISSUE TITLE
Service generator does not create symbol name for rosidl_typesupport_cpp

### ISSUE THREAD

**Issue by StefanFabian** [ros2/rosidl_typesupport#122]
## Bug report

**Required Info:**

- Operating System:
  - Ubuntu 20.04
- Installation type:
  - Binary
- Version or commit hash:
  - 1.2.0-1
- DDS implementation:
  - Fast-RTPS
- Client library (if applicable):
  - rclcpp

#### Steps to reproduce issue
Build a service definition

#### Expected behavior
There should be a symbol
`rosidl_typesupport_cpp__get_service_type_support_handle__[package_name]__srv__[interface_name]`

#### Actual behavior
There is no symbol
`rosidl_typesupport_cpp__get_service_type_support_handle__[package_name]__srv__[interface_name]`

#### Additional information

rosidl_typesupport_introspection_cpp contains the following:
```
#ifdef __cplusplus
extern "C"
{
#endif

ROSIDL_TYPESUPPORT_INTROSPECTION_CPP_PUBLIC
const rosidl_service_type_support_t *
ROSIDL_TYPESUPPORT_INTERFACE__SERVICE_SYMBOL_NAME(rosidl_typesupport_introspection_cpp, example_interfaces, srv, AddTwoInts)() {
  return ::rosidl_typesupport_introspection_cpp::get_service_type_support_handle<example_interfaces::srv::AddTwoInts>();
}

#ifdef __cplusplus
}
#endif

```
the same should probably be in rosidl_typesupport_cpp but is missing.

I don't know where exactly this is generated, hence, I can't make a PR for this.

**Comment by hidmic** [comment]
@StefanFabian good catch! The service typesupport symbol is missing from [`srv__type_support.cpp.em`](https://github.com/ros2/rosidl_typesupport/blob/edee0a7a51c9f4f17604bc3e4b4b8d70a4a2bc17/rosidl_typesupport_cpp/resource/srv__type_support.cpp.em#L155). A bugfix would be most welcome.

**Comment by StefanFabian** [comment]
I actually did already create PRs to fix this for foxy and galactic.
See #114 and #115.

**Comment by StefanFabian** [comment]
Any update on what's missing to get these PRs merged? @hidmic 
I can't release two large projects until that happens and it's really not a lot of code.
I understand that you guys are busy and have to make sure nothing will break but it's been 9 months since I made that PR.

**Comment by clalancette** [comment]
As far as I understand, this was solved with https://github.com/ros2/rosidl_typesupport/pull/143 and https://github.com/ros2/rosidl/pull/703 , so going ahead and closing this.  Feel free to reopen if you think this was a mistake.
